proj/src/main/webapp/static/signup.html

```diff
@@ -1,57 +1,60 @@
   <head>
     <meta charset="utf-8">
-    <title>Welcome</title>
+    <meta content="width=device-width, initial-scale=1" name="viewport" />
+
+    <title>camaraderie</title>
+
+    <!-- Styling -->
     <!-- Bootstrap: Latest compiled and minified CSS -->
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
-    <!-- Bootstrap: jQuery library -->
+    <!-- Bootstrap: jQuery library --> 
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
     <!-- Bootstrap: Latest compiled JavaScript -->
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
-    <link rel="stylesheet" href="../style.css">
     <!-- Font  -->
     <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="../style.css">
 
+    <!-- Firebase and other functionality scripts -->
     <!-- The core Firebase JS SDK is always required and must be listed first -->
     <script src="https://www.gstatic.com/firebasejs/7.15.4/firebase-app.js"></script>
-
     <!-- TODO: Add SDKs for Firebase products that you want to use
         https://firebase.google.com/docs/web/setup#available-libraries -->
     <script src="https://www.gstatic.com/firebasejs/7.15.4/firebase-analytics.js"></script>
-
     <script src="../app.js"></script>
-
     <script src="https://www.gstatic.com/firebasejs/7.15.4/firebase-auth.js"></script>
     <script src="https://www.gstatic.com/firebasejs/7.15.4/firebase-database.js"></script>
     <script src="https://www.gstatic.com/firebasejs/7.15.4/firebase-messaging.js"></script>
-  </head>
 
-  <body>
+  </head>
+  
+  <body> 
     <div class="container screen">
       <div class="row fullscreen">
-        <!-- title  desktop:left    mobile:none -->
-        <div class="col-sm-8 info-graphic">
-          <div class="header b-margin">
+        <!-- page:signup and signin   desktop:left    mobile:full screen -->
+        <div class="hide-on-mobile mobile-screen col-sm-8 info-graphic d-flex justify-content-center text-center">
+          <div class="header b-margin border border-warning">
             <h1>camaraderie</h1>
-            <h4>a spirit of friendly good-fellowship</h4>
+            <h4>meet friends • engage in communities</h4>
           </div>
           <div class="center-info">
             <div id="chat-image-containter">
-              <img id="chat-image"src="./images/friendsChatting.svg" alt="Friends chatting icon">
+              <img id="chat-image"src="./images/friendsChatting.svg" alt="Friends chatting icon" class="b-margin-3">
             </div>
             <div class="inblock-center-container">
-              <h3 class="black-bold-subtitle center-text"> meet friends <br> create communities</h3>
+              <h3 class="center-text" id="info-message"> spend less time finding chat rooms <br> and more time making friends</h3>
```

Comment:
I kind of like the other slogan a bit more because it was short and simple.

Reply:
You don't actually have to change this though.

Reply:
interesting, we can definitely talk about it in the team meeting tomorrow. i was thinking what it meant for a person that doesn't know what the app does but short and simple slogan are more impactful